<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snifferClass</class>
 <widget class="QMainWindow" name="snifferClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sniffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableWidget" name="tableWidget"/>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1134</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuproject">
    <property name="title">
     <string>project</string>
    </property>
   </widget>
   <widget class="QMenu" name="menurun">
    <property name="title">
     <string>run</string>
    </property>
    <addaction name="actionrun"/>
    <addaction name="actionempty"/>
   </widget>
   <addaction name="menuproject"/>
   <addaction name="menurun"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionrun">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/new/prefix1/start.png</normaloff>:/new/prefix1/start.png</iconset>
   </property>
   <property name="text">
    <string>runandstop</string>
   </property>
  </action>
  <action name="actionempty">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/new/prefix1/empty.png</normaloff>:/new/prefix1/empty.png</iconset>
   </property>
   <property name="text">
    <string>empty</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sniffer.qrc"/>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
